tests/unit/Newsletter/Renderer/StylesHelperTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+namespace MailPoet\Test\Newsletter;
+
+use MailPoet\Newsletter\Renderer\StylesHelper;
+
+class StylesHelperTest extends \MailPoetTest {
+
+  function __construct() {
```

Comment:
Why did you use `__construct` instead of `before` like we do in all our tests?

Reply:
I have just looked at https://github.com/mailpoet/mailpoet/blob/master/tests/unit/Newsletter/RendererTest.php
and did the same here.